packages/application-extension/schema/commands.json

```
@@ -294,7 +294,7 @@
                   "rank": 11
                 },
                 {
-                  "command": "application:toggle-side-tabbar",
+                  "command": "application:toggle-sidebar-widget",
```

```suggestion
                  "command": "application:toggle-side-tabbar",
```

This is the menu command, it should not be changed.